distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1233,6 +1233,125 @@ public void testValidationFiltering() throws Exception {
+                    id2))));
+
+            // Search for all entries that have no validation warnings or errors
+            query = new CswQueryBuilder().addPropertyIsNullAttributeFilter(VALIDATION_WARNINGS)
+                    .getQuery();
+            response = given().header("Content-Type", MediaType.APPLICATION_XML)
+                    .body(query)
+                    .post(CSW_PATH.getUrl())
+                    .then();
+            // Assert Metacard1 is in results AND not Metacard2
+            response.body(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id1)));
+            response.body(not(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id2))));
+
+            //Search for all entries that have validation-warnings from sample-validator or no validation warnings
+            //Only search that will actually return all entries
+
+            query = new CswQueryBuilder().addAttributeFilter(PROPERTY_IS_EQUAL_TO,
+                    VALIDATION_WARNINGS,
+                    "*")
+                    .addPropertyIsNullAttributeFilter(VALIDATION_WARNINGS)
+                    .addLogicalOperator(OR)
+                    .getQuery();
+
+            response = given().header("Content-Type", MediaType.APPLICATION_XML)
+                    .body(query)
+                    .post(CSW_PATH.getUrl())
+                    .then();
+            // Assert Metacard1 and NOT metacard2 is in results
+            response.body(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id1)));
+            response.body(not(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id2))));
+
+            // Search for all metacards that have validation-warnings
+            query = new CswQueryBuilder().addAttributeFilter(PROPERTY_IS_EQUAL_TO,
+                    VALIDATION_WARNINGS,
+                    "*")
+                    .getQuery();
+
+            response = given().header("Content-Type", MediaType.APPLICATION_XML)
+                    .body(query)
+                    .post(CSW_PATH.getUrl())
+                    .then();
+            // Assert Metacard1 and metacard2 are NOT in results
+            response.body(not(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id1))));
+            response.body(not(hasXPath(String.format(
+                    "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
+                    id2))));
+
+            deleteMetacard(id1);
+        } finally {
+            getServiceManager().stopFeature(true,
+                    "catalog-plugin-metacard-validation",
+                    "catalog-security-filter");
```

Comment:
catalog-security-filter is on by default and needs to remain on, this should be removed.